WalletWasabi.Fluent/ViewModels/CoinSelection/Core/Cells/AnonymityScoreCellViewModel.cs

```diff
@@ -0,0 +1,25 @@
+using WalletWasabi.Fluent.ViewModels.Wallets.Advanced.WalletCoins;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinSelection.Core.Cells;
+
+public class AnonymityScoreCellViewModel : ViewModelBase
+{
+	public AnonymityScoreCellViewModel(WalletCoinViewModel coin)
+	{
+		PrivacyScore = coin.AnonymitySet;
+		PrivacyLevel = coin.GetPrivacyLevel();
+		IsPrivate = PrivacyLevel == PrivacyLevel.Private;
+		IsSemiPrivate = PrivacyLevel == PrivacyLevel.SemiPrivate;
+		IsNonPrivate = PrivacyLevel == PrivacyLevel.NonPrivate;
+	}
+
+	public bool IsSemiPrivate { get; }
+
+	public bool IsNonPrivate { get; }
+
+	public bool IsPrivate { get; }
+
+	public PrivacyLevel PrivacyLevel { get; }
```

Comment:
This is to apply a different color to each value. I created a converter for this, but it was ugly and didn't work with dynamic resources. Also, I tried with Behaviors, but the resulting code was pretty cumbersome.

It would be nice to have a better way to do this.